template-parts/homepage-landing.php

```diff
@@ -10,13 +10,13 @@
     <div class="swiper-wrapper">
         <!-- Slides -->
         <div class="swiper-slide">
-        	<div class="content">Since 2005, <span style="color:#CE2727">8,730 deaths</span> have been reported in Texas custody.</div>
+        	<div class="content">Since 2005, <span style="color:#CE2727"><span id="cdrTotal"></span> deaths</span> have been reported in Texas custody.</div>
```

Comment:
It looks like we could also make the dates dynamic here too. Those are in the dataset files as well.